CONTRIBUTING.adoc

```diff
@@ -66,9 +66,9 @@ export OS_OUTPUT_GOPATH=1
 
         $ mkdir -p $GOPATH/src/github.com/openshift
         $ cd $GOPATH/src/github.com/openshift
-        $ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+        $ git clone https://github.com/<forkid>/origin  # Replace <forkid> with the your github id
```

Comment:
unfortunately, I think ssh and git protocols not working are probably due to your proxy. I think most devs will have sshkey auth set up to github, not username/password auth, so I'd rather keep the docs using either the ssh (`git@github.com:...`) or git (`git://...`) protocols